.github/workflows/ci.yml

```diff
@@ -103,6 +103,10 @@ jobs:
         uses: "actions/checkout@v4"
       - name: "Setup environment"
         uses: "networktocode/gh-action-setup-poetry-environment@v6"
+      - name: "Constrain Nautobot version and regenerate lock file"
+        env:
+          INVOKE_NAUTOBOT_DEV_EXAMPLE_LOCAL: "true"
```

Comment:
Looks like I messed up the cookie here :(

Reply:
I just saw your updated PR, not sure what will happen with this PR once that gets released, I guess a good time to try!